examples/fast-api/fast_api.py

```
@@ -0,0 +1,138 @@
+    start_time = time.time()
+
+    # Log request
+    await logger.log(
+        "info",
+        f"{request.method} {request.url.path}",
+        {
+            "method": request.method,
+            "path": request.url.path,
+            "client_ip": request.client.host if request.client else None,
+        },
+    )
```

Pylint (3.3.7)

[error] Too many positional arguments for method call

(E1121)